### PR TITLE
fix: broken refragmentation

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

File: ldes-server-infra-postgres/postgres-ingest-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/postgres/batch/MemberItemReader.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public JdbcPagingItemReader<FragmentationMember> refragmentEventStream(@Value("#
 				.dataSource(dataSource)
 				.rowMapper(new MemberRowMapper())
 				.queryProvider(refragmentQuery())
-				.parameterValues(Map.of("viewName", jobParameters.get("viewName")))
+				.parameterValues(Map.of("viewName", jobParameters.get("viewName"), "collectionName", jobParameters.get("collectionName")))
```
Comment (jobulcke) [inline ldes-server-infra-postgres/postgres-ingest-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/postgres/batch/MemberItemReader.java:39]: I never really understood the purpose of creating a new map with the same entries as the provided jobParameters map

File: ldes-server-integration-test/src/test/resources/application-postgres-test.yml [hunk 2]
```diff
@@ -19,4 +19,4 @@ spring:
 
 logging:
   level:
-    be.vlaanderen.informatievlaanderen: DEBUG
\ No newline at end of file
+    org.springframework.batch.core: ERROR
```
Comment (jobulcke) [inline ldes-server-integration-test/src/test/resources/application-postgres-test.yml:22]: Not even warns?

File: ldes-server-integration-test/src/test/resources/features/fragmentation/fragmentation.feature [hunk 3]
```diff
@@ -21,6 +21,27 @@ Feature: LDES Server Fragmentation
...
+    And this fragment is immutable
+    When I fetch the next fragment through the first "Relation"
+    Then this fragment only has 1 "Relation" relation
+    And this fragment is immutable
+    When I fetch the next fragment through the first "Relation"
+    And this fragment contains <restCount> members
+    And this fragment is mutable
+    And this fragment has no relations
+    Examples:
+      | eventStreamDescriptionFile                                      | viewDescriptionFile                                                       | template                                   | collection            | ingestedMemberCount | restCount |
```
Comment (jobulcke) [inline ldes-server-integration-test/src/test/resources/features/fragmentation/fragmentation.feature:42]: No example for state ingestion?

**Review by jobulcke — APPROVED** [review]

File: ldes-server-integration-test/src/test/resources/features/fragmentation/fragmentation.feature (hunk 3 above)
Comment (Yalz) [reply ldes-server-integration-test/src/test/resources/features/fragmentation/fragmentation.feature:42]: honestly it doesn't really matter. I don't want to clog up our ITs with just more spam.. 
This test mainly just checks if pages are made after ingestion took place. state vs version object ingestion doesn't matter at that point anymore

**Pull request merged**